<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Using the CNV information to identify molecular subtypes informative for prognosis and treatment in pancreatic adenocarcinoma (PAAD) of a Chinese cohort</title>
  <meta name="description" content="Using the CNV information to identify molecular subtypes informative for prognosis and treatment in pancreatic adenocarcinoma (PAAD) of a Chinese cohort" />
  <meta name="generator" content="bookdown 0.22 and GitBook 2.6.7" />

  <meta property="og:title" content="Using the CNV information to identify molecular subtypes informative for prognosis and treatment in pancreatic adenocarcinoma (PAAD) of a Chinese cohort" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Using the CNV information to identify molecular subtypes informative for prognosis and treatment in pancreatic adenocarcinoma (PAAD) of a Chinese cohort" />
  
  
  



<meta name="date" content="2021-11-26" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  

<link rel="next" href="mutational-landscape.html"/>
<script src="libs/header-attrs-2.8/header-attrs.js"></script>
<script src="libs/jquery-3.5.1/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>
<script src="libs/htmlwidgets-1.5.3/htmlwidgets.js"></script>
<link href="libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="libs/datatables-binding-0.18/datatables.js"></script>
<link href="libs/dt-core-1.10.20/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="libs/dt-core-1.10.20/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="libs/dt-core-1.10.20/js/jquery.dataTables.min.js"></script>
<link href="libs/dt-ext-buttons-1.10.20/css/buttons.dataTables.min.css" rel="stylesheet" />
<script src="libs/dt-ext-buttons-1.10.20/js/dataTables.buttons.min.js"></script>
<script src="libs/dt-ext-buttons-1.10.20/js/buttons.flash.min.js"></script>
<script src="libs/dt-ext-buttons-1.10.20/js/buttons.html5.min.js"></script>
<script src="libs/dt-ext-buttons-1.10.20/js/buttons.colVis.min.js"></script>
<script src="libs/dt-ext-buttons-1.10.20/js/buttons.print.min.js"></script>
<link href="libs/dt-plugin-searchhighlight-1.10.20/source.css" rel="stylesheet" />
<script src="libs/dt-plugin-searchhighlight-1.10.20/jquery.highlight.js"></script>
<script src="libs/dt-plugin-searchhighlight-1.10.20/source.min.js"></script>
<link href="libs/crosstalk-1.1.1/css/crosstalk.css" rel="stylesheet" />
<script src="libs/crosstalk-1.1.1/js/crosstalk.min.js"></script>




</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./index.html">PAAD</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Overview</a>
<ul>
<li class="chapter" data-level="1.1" data-path="index.html"><a href="index.html#motivation"><i class="fa fa-check"></i><b>1.1</b> Motivation</a></li>
<li class="chapter" data-level="1.2" data-path="index.html"><a href="index.html#what-has-known-in-literature"><i class="fa fa-check"></i><b>1.2</b> What has known in literature</a></li>
<li class="chapter" data-level="1.3" data-path="index.html"><a href="index.html#what-has-done-this-study"><i class="fa fa-check"></i><b>1.3</b> What has done this study</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="mutational-landscape.html"><a href="mutational-landscape.html"><i class="fa fa-check"></i><b>2</b> Mutational landscape</a></li>
<li class="chapter" data-level="3" data-path="cnv-and-prognosis.html"><a href="cnv-and-prognosis.html"><i class="fa fa-check"></i><b>3</b> CNV and prognosis</a>
<ul>
<li class="chapter" data-level="3.1" data-path="cnv-and-prognosis.html"><a href="cnv-and-prognosis.html#differential-cnv"><i class="fa fa-check"></i><b>3.1</b> Differential CNV</a></li>
<li class="chapter" data-level="3.2" data-path="cnv-and-prognosis.html"><a href="cnv-and-prognosis.html#prognosis"><i class="fa fa-check"></i><b>3.2</b> Prognosis</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="molecular-subtypes.html"><a href="molecular-subtypes.html"><i class="fa fa-check"></i><b>4</b> Molecular subtypes</a>
<ul>
<li class="chapter" data-level="4.1" data-path="molecular-subtypes.html"><a href="molecular-subtypes.html#subtypes-and-prognosis"><i class="fa fa-check"></i><b>4.1</b> Subtypes and prognosis</a></li>
<li class="chapter" data-level="4.2" data-path="molecular-subtypes.html"><a href="molecular-subtypes.html#repair-deficient-patients"><i class="fa fa-check"></i><b>4.2</b> Repair-deficient patients</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="code-availability.html"><a href="code-availability.html"><i class="fa fa-check"></i><b>5</b> Code availability</a></li>
<li class="divider"></li>
<li>Email: shenby@shsmu.edu.cn</li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Using the CNV information to identify molecular subtypes informative for prognosis and treatment in pancreatic adenocarcinoma (PAAD) of a Chinese cohort</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="header">
<h1 class="title">Using the CNV information to identify molecular subtypes informative for prognosis and treatment in pancreatic adenocarcinoma (PAAD) of a Chinese cohort</h1>
<p class="author"><em><div class="line-block">Qian Zhan, Chenlei Wen, Yi Zhao, Lu Fang, Yangbing Jin, Zehui Zhang, Siyi Zou, Fanlu Li, Ying Yang, Lijia Wu, Jiabin Jin, Xiongxiong Lu, Junjie Xie, Dongfeng Cheng, Zhiwei Xu, Jun Zhang, Jiancheng Wang, XiaXing Deng, Hao Chen, Chenghong Peng, Hongwei Li, <a href="mailto:zhang.henghui@genecast.com.cn">Henghui Zhang</a>, <a href="mailto:fh12355@rjh.com.cn">Hai Fang</a>, <a href="mailto:wcf11956@rjh.com.cn">Chaofu Wang</a>, <a href="mailto:shenby@shsmu.edu.cn">Baiyong Shen</a><br />
<br />
Department of General Surgery, Pancreatic Disease Center, Research Institute of Pancreatic Diseases, Ruijin Hospital, Shanghai Jiao Tong University School of Medicine, Shanghai, China<br />
State Key Laboratory of Oncogenes and Related Genes, National Research Center for Translational Medicine (Shanghai), Shanghai, China<br />
Genecast Biotechnology Co., Ltd, Wuxi, China<br />
Shanghai Institute of Hematology, State Key Laboratory of Medical Genomics, National Research Center for Translational Medicine (Shanghai), Ruijin Hospital, Shanghai Jiao Tong University School of Medicine, Shanghai, China<br />
Department of Pathology, Ruijin Hospital, Shanghai Jiao Tong University School of Medicine, Shanghai, China</div></em></p>
<p class="date" style="margin-top: 1.5em;"><em>2021-11-26</em></p>
</div>
<div id="index" class="section level1" number="1">
<h1><span class="header-section-number">Section 1</span> Overview</h1>
<div id="motivation" class="section level2" number="1.1">
<h2><span class="header-section-number">1.1</span> Motivation</h2>
<p>According to <a href="https://pubmed.ncbi.nlm.nih.gov/26808342">cancer statistics in China, 2015</a>, PAAD is one of the most aggressive and deadly cancers in China, with an estimate of 90,100 new cases and 79,400 new deaths. We recognize that the high mortality of PAAD in China can be traced back to the lack of comprehensive molecular subtyping (and prognostic models to evaluate risk levels of relapse). We also realize that the current histopathological classifications for PAAD are of limited use to guide clinical decision-making. Therefore, there is an unmet need to identify new biomarkers for molecular subtyping in PAAD.</p>
</div>
<div id="what-has-known-in-literature" class="section level2" number="1.2">
<h2><span class="header-section-number">1.2</span> What has known in literature</h2>
<p>The current stratification for PAAD patients is mainly based on:</p>
<ul>
<li><p>the mutation profiles of genes, including homologous recombination repair (HRR) genes (see <a href="https://pubmed.ncbi.nlm.nih.gov/32855305">paper</a>), and highly mutated genes (such as KRAS, TP53, CDKN2A and SMAD4; see <a href="https://pubmed.ncbi.nlm.nih.gov/30836094">paper</a>); and</p></li>
<li><p>the RNA-seq expression data to estimate immune cytolytic activity (see <a href="https://pubmed.ncbi.nlm.nih.gov/28007776">paper</a>).</p></li>
</ul>
</div>
<div id="what-has-done-this-study" class="section level2" number="1.3">
<h2><span class="header-section-number">1.3</span> What has done this study</h2>
<ul>
<li><p>We report the <strong><a href="mutational-landscape.html#mutational-landscape">mutational landscape</a></strong> of 608 PAAD patients, the largest cohort ever in China, generating the most comprehensive resource on genetic alternations (covering somatic mutations, pathogenic germline variants, copy number variations, and well-known genomic markers, such as tumor mutational burden, copy number instability and somatic signatures). Based on the information of point mutations in either HRR genes or highly mutated genes, as mentioned above (<a href="index.html#what-has-known-in-literature">What has known in literature</a>), we find no associations with prognosis.</p></li>
<li><p>We are the first to report that the poor prognosis is associated with amplification of genes involved in DNA repair and receptor tyrosine kinase (RTK) related signaling pathways, establishing the clinical link between <strong><a href="cnv-and-prognosis.html#cnv-and-prognosis">CNV and prognosis</a></strong> in PAAD.</p></li>
<li><p>Using the copy number variations (CNV) information of genes involved in DNA repair and RTK related signaling pathways, we are able to stratify patients into four <strong><a href="molecular-subtypes.html#molecular-subtypes">molecular subtypes</a></strong> (namely, <code>repair-deficient</code>, <code>proliferation-active</code>, <code>repair-proficient</code> and <code>repair-enhanced</code> subtypes) that differ in prognosis.</p></li>
<li><p>We suggest the treatment management for PAAD patients (see <strong>the figure</strong> below): repair-deficient and proliferation-active subtypes are better suited for DNA damage therapies, while immunotherapy highly recommended for repair-proficient and repair-enhanced subtypes.</p></li>
</ul>
<p><img src="PAAD_files/figure-html/management-1.png" width="80%" style="display: block; margin: auto;" /></p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>

<a href="mutational-landscape.html" class="navigation navigation-next navigation-unique" aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": false,
"twitter": false,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": null,
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"toc": {
"collapse": "subsection",
"scroll_highlight": true
},
"toolbar": {
"position": "fixed"
},
"search": false,
"info": false
});
});
</script>

</body>

</html>
