<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta name="viewport" content="width=device-width, initial-scale=1" />
<meta property="og:title" content="Using the CNV information to identify molecular subtypes in pancreatic adenocarcinoma (PAAD)" />
<meta property="og:type" content="book" />








<meta name="description" content="Using the CNV information to identify molecular subtypes in pancreatic adenocarcinoma (PAAD)">

<title>Using the CNV information to identify molecular subtypes in pancreatic adenocarcinoma (PAAD)</title>

<link href="libs/tufte-css-2015.12.29/tufte-fonts.css" rel="stylesheet" />
<link href="libs/tufte-css-2015.12.29/tufte-background.css" rel="stylesheet" />
<link href="libs/tufte-css-2015.12.29/tufte-italics.css" rel="stylesheet" />
<link href="libs/tufte-css-2015.12.29/tufte.css" rel="stylesheet" />





</head>

<body>



<div class="row">
<div class="col-sm-12">
<!--bookdown:toc:end-->
<!--bookdown:toc:start-->
</div>
</div>
<div class="row">
<div class="col-sm-12">
<div id="header">
<h1 class="title">Using the CNV information to identify molecular subtypes in pancreatic adenocarcinoma (PAAD)</h1>
<h4 class="author"><em><div style="white-space: pre-line;">Qian Zhan, Chenlei Wen, Yi Zhao, Lu Fang, Yangbing Jin, Zehui Zhang, Siyi Zou, Fanlu Li, Ying Yang, Lijia Wu, Jiabin Jin, Xiongxiong Lu, Junjie Xie, Dongfeng Cheng, Zhiwei Xu, Jun Zhang, Jiancheng Wang, XiaXing Deng, Hao Chen, Chenghong Peng, Hongwei Li, Henghui Zhang, Hai Fang, Chaofu Wang, <a href="mailto:shenby@shsmu.edu.cn">Baiyong Shen</a>

Department of General Surgery, Pancreatic Disease Center, Research Institute of Pancreatic Diseases, Ruijin Hospital, Shanghai Jiao Tong University School of Medicine, 197 Ruijin Er Road, Shanghai, 200025, China</div></em></h4>
</div>
<div id="index" class="section level1">
<h1><span class="header-section-number">Section 1</span> Overview</h1>
<div id="motivation" class="section level2">
<h2><span class="header-section-number">1.1</span> <strong>Motivation</strong></h2>
<p>According to the <a href="https://pubmed.ncbi.nlm.nih.gov/26808342">cancer statistics</a>, PAAD is one of the most aggressive and deadly cancers in China, with an estimate of 90,100 new cases and 79,400 new deaths. We recognize that the high mortality of PAAD in China can be traced back to the lack of comprehensive molecular subtyping (and prognostic models to evaluate risk levels of relapse), and that the current histopathological classifications are of limited use to guide clinical decision-making. Therefore, there is an unmet need to identify new biomarkers for molecular subtyping in PAAD.</p>
</div>
<div id="what-has-known-in-literature" class="section level2">
<h2><span class="header-section-number">1.2</span> <strong>What has known in literature</strong></h2>
<p>The current stratification for PAAD patients is mainly based on: i) the mutation profiles of genes, including homologous recombination repair (<a href="https://pubmed.ncbi.nlm.nih.gov/32855305">HRR</a>) genes, and <a href="https://pubmed.ncbi.nlm.nih.gov/30836094">highly mutated</a> genes (such as KRAS, TP53, CDKN2A and SMAD4); and ii) the RNA-seq expression data to estimate <a href="https://pubmed.ncbi.nlm.nih.gov/28007776">immune cytolytic activity</a>.</p>
</div>
<div id="what-has-done-this-study" class="section level2">
<h2><span class="header-section-number">1.3</span> <strong>What has done this study</strong></h2>
<ul>
<li><p>We report the mutational landscape of 608 PAAD patients, the largest cohort ever in China, generating the most comprehensive resource on genetic alternations (somatic mutations, pathogenic germline variants, copy number variations, and well-known genomic markers, such as TMB, copy number instability and somatic signatures).</p></li>
<li><p>We are the first to report that the poor prognosis is associated with amplification of genes involved in DNA repair and RTK related signaling pathways.</p></li>
<li><p>Using the information of copy number variations (CNV), we are able to stratify patients into four molecular subtypes (namely, repair-deficient, proliferation-active, repair-proficient and repair-enhanced subtypes) that differed in prognosis.</p></li>
<li><p>We show that repair-deficient and proliferation-active subtypes are better suited for DNA damage therapies, while immunotherapy highly recommended for repair-proficient and repair-enhanced subtypes.</p></li>
<li><p>We construct a prognostic model incorporating the information on CNV of DNA repair and RTK related genes, and illustrate the use of the constructed model in distinguishing patients with high and low risk of relapse.</p></li>
<li><p>We reinforce the prognostic value of our identified genes in DNA repair and RTK related signaling pathways by analyzing publicly available datasets.</p></li>
</ul>
</div>
</div>
<div id="mutational-landscape" class="section level1">
<h1><span class="header-section-number">Section 2</span> Mutational landscape</h1>
<p>The top ranked targets (~150) were highly enriched for genes involved in TRAF6- or TNFR1-induced NF-kappa B activation, including BIRC2/3, IKBKB/G, NFKB1/2, NFKBIA/B, TRAF1/2/6, MYD88 and TNF.</p>
<p><img src="PAAD_files/figure-html/pathway-1.png" width="100%"  style="display: block; margin: auto;" /></p>

</div>
<p style="text-align: center;">
</p>
</div>
</div>



</body>
</html>
