---
documentclass: book
classoption: oneside
title: "Using the CNV information to identify molecular subtypes in pancreatic adenocarcinoma (PAAD)"
author: |
 | Qian Zhan, Chenlei Wen, Yi Zhao, Lu Fang, Yangbing Jin, Zehui Zhang, Siyi Zou, Fanlu Li, Ying Yang, Lijia Wu, Jiabin Jin, Xiongxiong Lu, Junjie Xie, Dongfeng Cheng, Zhiwei Xu, Jun Zhang, Jiancheng Wang, XiaXing Deng, Hao Chen, Chenghong Peng, Hongwei Li, Henghui Zhang, Hai Fang, Chaofu Wang, [Baiyong Shen](mailto:shenby@shsmu.edu.cn) 
 | 
 | Department of General Surgery, Pancreatic Disease Center, Research Institute of Pancreatic Diseases, Ruijin Hospital, Shanghai Jiao Tong University School of Medicine, 197 Ruijin Er Road, Shanghai, 200025, China
vignette: >
 %\VignetteIndexEntry{PAAD Booklet}
 %\VignetteEngine{knitr::rmarkdown}
 %\VignetteEncoding{UTF-8}
output: 
  BiocStyle::html_document:
        toc: true
        toc_depth: 3
        number_sections: true
        theme: journal
        highlight: monochrome
        toc_float: true
        code_folding: hide
---

```{r, include=FALSE}
knitr::opts_chunk$set(collapse=TRUE, comment="##")
knitr::opts_chunk$set(dpi=300)
knitr::opts_chunk$set(cache=FALSE)
knitr::opts_chunk$set(echo=TRUE)
knitr::opts_chunk$set(warning=FALSE)
knitr::opts_chunk$set(message=FALSE)
knitr::opts_chunk$set(fig_cap="yes")
knitr::opts_chunk$set(eval=F)
```


# Overview {#index}

## Motivation

According to [cancer statistics in China, 2015](https://pubmed.ncbi.nlm.nih.gov/26808342), PAAD is one of the most aggressive and deadly cancers in China, with an estimate of 90,100 new cases and 79,400 new deaths. We recognize that the high mortality of PAAD in China can be traced back to the lack of comprehensive molecular subtyping (and prognostic models to evaluate risk levels of relapse). We also realize that the current histopathological classifications for PAAD are of limited use to guide clinical decision-making. Therefore, there is an unmet need to identify new biomarkers for molecular subtyping in PAAD.

## What has known in literature

The current stratification for PAAD patients is mainly based on: 

- the mutation profiles of genes, including homologous recombination repair (HRR) genes (see [paper](https://pubmed.ncbi.nlm.nih.gov/32855305)), and highly mutated genes (such as KRAS, TP53, CDKN2A and SMAD4; see [paper](https://pubmed.ncbi.nlm.nih.gov/30836094)); and

- the RNA-seq expression data to estimate immune cytolytic activity (see [paper](https://pubmed.ncbi.nlm.nih.gov/28007776)).

## What has done this study

- We report the **[mutational landscape]** of 608 PAAD patients, the largest cohort ever  in China, generating the most comprehensive resource on genetic alternations (covering somatic mutations, pathogenic germline variants, copy number variations, and well-known genomic markers, such as tumor mutational burden, copy number instability and somatic signatures). Based on the information of point mutations in either HRR genes or highly mutated genes, as mentioned above ([What has known in literature]), we find no associations with prognosis. 

- We are the first to report that the poor prognosis is associated with amplification of genes involved in DNA repair and receptor tyrosine kinase (RTK) related signaling pathways, supporting the clinical link between **[CNV and prognosis]** in PAAD.

- Using the information of copy number variations (CNV), we are able to stratify patients into four molecular subtypes (namely, `repair-deficient`, `proliferation-active`, `repair-proficient` and `repair-enhanced` subtypes) that differ in prognosis. 

- We show that repair-deficient and proliferation-active subtypes are better suited for DNA damage therapies, while immunotherapy highly recommended for repair-proficient and repair-enhanced subtypes. 

- We construct a prognostic model incorporating the information on CNV of DNA repair and RTK related genes, and illustrate the use of the constructed model in distinguishing patients with high and low risk of relapse. 

- We reinforce the prognostic value of our identified genes in DNA repair and RTK related signaling pathways by analyzing publicly available datasets.

```{r, eval=FALSE, echo=FALSE}
cd ~/Sites/XGR/PAAD/BOOKLET
################################################
R
library(tidyverse)

## load
load("now.PAAD_booklet.RData")

## save
save(list=ls(), file="now.PAAD_booklet.RData", compress="xz")

## bookdown::render_book + bookdown::gitbook
## use configuration in '_output.yml' and '_bookdown.yml'
bookdown::render_book("now.PAAD_booklet.Rmd", c("bookdown::gitbook","bookdown::pdf_book","bookdown::tufte_html_book")[1])

q('no')

####################################################

##########################
https://23verse.github.io/PAAD
##########################
## bookdown::render_book + bookdown::gitbook
cd ~/Sites/SVN/github/PAAD
# delete previous *.html *.png *.css *.js *.ttf *.Rmd files *.yml files
find ./ -type f -name '*.html' -o -type f -name '*.png' -o -type f -name '*.css' -o -type f -name '*.js' -o -type f -name '*.ttf' -o -type f -name '*.Rmd' -o -type f -name '*.yml' -o -type f -name '*.RDS' -o -type f -name '*.r' | xargs rm -rf
############
# copy the whole package from ~/Sites/XGR/PAAD/BOOKLET/PAAD2booklet to ~/Sites/SVN/github/PAAD
cp -rf ~/Sites/XGR/PAAD/BOOKLET/PAAD2booklet/* ./
# copy/rename ~/Sites/XGR/PAAD/BOOKLET/now.PAAD_booklet.Rmd to ~/Sites/SVN/github/PAAD/PAAD.Rmd
cp -rf ~/Sites/XGR/PAAD/BOOKLET/now.PAAD_booklet.Rmd ./PAAD.Rmd
cp -rf ~/Sites/XGR/PAAD/BOOKLET/_bookdown.yml ./_bookdown.yml
cp -rf ~/Sites/XGR/PAAD/BOOKLET/_output.yml ./_output.yml
# add new *.html *.png *.css *.js *.ttf *.Rmd *.yml files
git status | grep 'modified' | awk '{print $2}' | xargs git add
git status | grep 'deleted' | awk '{print $2}' | xargs git rm
find ./ -type f -name '*.html' -o -type f -name '*.png' -o -type f -name '*.css' -o -type f -name '*.js' -o -type f -name '*.ttf' -o -type f -name '*.Rmd' -o -type f -name '*.yml' -o -type f -name '*.RDS' -o -type f -name '*.r' | xargs git add
git add --all
git commit -m 'Update PAAD'
git push origin main
##########################

####################################################
## GitHub Pages

One site per account but unlimited projects can be created from https://pages.github.com:

> **One site `https://github.com/23verse/23verse.github.io`**

# visit https://github.com/new (enter "23verse.github.io" -> Initialize this repository with a README)
# clone into the local machine 
cd ~/Sites/SVN/github
git clone git@github.com:23verse/23verse.github.io
cd ~/Sites/SVN/github/23verse.github.io
git pull origin master
## https://github.com/23verse/23verse.github.io/settings/access: add "hfang-bristol"
echo "Hello 23verse" > index.html
git add --all
git commit -m 'Initial commit'
git push origin master
# Alive (https://23verse.github.io)

> **A project `https://github.com/23verse/PAAD`**

# visit https://github.com/new (enter "PAAD" -> Initialize this repository with a README)
## https://github.com/23verse/PAAD/settings/access: add "hfang-bristol"
# clone into the local machine 
cd ~/Sites/SVN/github
git clone git@github.com:23verse/PAAD.git
cd ~/Sites/SVN/github/PAAD
git pull origin main
echo "Hello PAAD" > index.html
git add --all
git commit -m 'Initial commit'
git push origin main
# visit https://github.com/23verse/PAAD/settings -> GitHub Pages (main/root)
# Alive (https://23verse.github.io/PAAD): Molecular subtyping in PAAD
####################################################

```

```{r, eval=F, echo=F}
library(tidyverse)
RData.location <- "~/Sites/XGR/PAAD/BOOKLET"
```

```{r, eval=F, echo=F}
## PAAD_priority.txt.gz
cp ~/Databases/grant_app/jobs/Oxford/Oxford_Wellcome_Trust_Centre_for_Human_Genetics/ULTRA-DD/MDPI/PAAD/PAAD_priority.txt.gz ~/Sites/XGR/PAAD/BOOKLET

## PAAD_reactome.txt
cp ~/Databases/grant_app/jobs/Oxford/Oxford_Wellcome_Trust_Centre_for_Human_Genetics/ULTRA-DD/MDPI/PAAD/PAAD_reactome.txt ~/Sites/XGR/PAAD/BOOKLET

## PAAD_subg_nodes.txt
cp ~/Databases/grant_app/jobs/Oxford/Oxford_Wellcome_Trust_Centre_for_Human_Genetics/ULTRA-DD/MDPI/PAAD/PAAD_subg_nodes.txt ~/Sites/XGR/PAAD/BOOKLET

## Crosstalk.approved_phased.txt
cp ~/Databases/grant_app/jobs/Oxford/Oxford_Wellcome_Trust_Centre_for_Human_Genetics/ULTRA-DD/MDPI/PAAD/Crosstalk.approved_phased.txt ~/Sites/XGR/PAAD/BOOKLET

```


# Mutational landscape

We carried out a cohort study involving 608 PAAD patients in China and, for each patient, profiled genetic alterations including somatic mutations, pathogenic germline variants and CNV along with several genomic markers, such as tumor mutational burden (TMB), copy number instability (CNI) and somatic signatures. **From this Chinese cohort we identified a list of frequently mutated genes, with the top 30 illustrated below.**

```{r landscape, fig.cap="", fig.width=7.5, fig.height=3, echo=FALSE, eval=T, fig.align="center",out.width="100%"}
library(png)
library(grid)
img <- readPNG("booklet.PAAD.landscape.png")
grid.raster(img)
```

# CNV and prognosis

We classified 608 PAAD patients into two groups according to CNV: the first group (CNV-G1 with 321 patients) versus the second group (CNV-G2 with 287 patients). We performed differential CNV analysis identifying 115 genes with higher CNV in CNV-G1 than in CNV-G2, and 255 genes with higher CNV in CNV-G2. These 255 genes with higher CNV in CNV-G2 are mostly of functional relevance to DNA repair, including 11 HRR genes (ATM, ATR, BLM, BRCA1/2, BRIP1, CHEK2, PALB2 and RAD51B/51C/51D). These analyses reveal that CNV-G2 patients seemed to obtain an enhanced function of DNA repair (likely HRR), while CNV-G1 patients possessed more depleted DNA repair genes. For convenience, we renamed CNV-G2 as 'repair-proficient', and CNV-G1 as 'repair-deficient'. **As shown in the left panel of the figure below, `patients in CNV-G2 (repair-proficient)` had a worse prognosis than `patients in CNV-G1 (repair-deficient)`.**

```{r prognosis, fig.cap="", fig.width=11.8, fig.height=3, echo=FALSE, eval=T, fig.align="center",out.width="100%"}
library(png)
library(grid)
img <- readPNG("booklet.PAAD.prognosis.png")
grid.raster(img)
```

Given the fact that patients in CNV-G1 (repair-deficient) formed a large group in number, we partitioned these patients into two subgroups: one subgroup with high CNV score, and the other subgroup with low CNV score. Comparing repair-deficient patients with high or low CNV score, we identified 203 genes with differential CNV. Genes with higher CNV (in the patient subgroup with higher CNV score) were largely involved in the RTK related signaling pathways. **As shown in the right panel of the figure above, worse prognosis was significantly associated with `the subgroup with high CNV score` rather than `the subgroup with low CNV score`.**